apps/web/src/server/applications/create/key-dates/applications-create-key-dates.validators.js

```diff
@@ -45,7 +45,7 @@ export const validateApplicationsCreateKeyDates = createValidator(
 			const { submissionInternalDay, submissionInternalMonth, submissionInternalYear } = req.body;
 			const submissionInternalDate = new Date(
 				submissionInternalYear,
-				submissionInternalMonth,
+				submissionInternalMonth - 1,
```

Comment:
It seems that earlier in the file we are using `Number.parseInt(submissionInternalMonth, 10)` to do the same thing? can we potentially standardise here?

Reply:
can do.  do we need to parseInt on the year and day too, to be consistent here?

Reply:
Not sure, it doesn't seem like the day and year need to undergo the same treatment.

Reply:
ah, because I need to make a calculation on the month value (changing from 1-12 to 0-11 in this case!).  OK, I will just do it for month.